pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -849,13 +852,18 @@ func verifyStatus(ctx *domainContext, status *types.DomainStatus) {
 				err := fmt.Errorf("one of the %s tasks has crashed (%v)", status.Key(), err)
 				log.Errorf(err.Error())
 				status.SetErrorNow("one of the application's tasks has crashed - please restart application instance")
```

Comment:
Do we ever hit this broken case when the app instance is shutdown or halted from inside?

Reply:
I assume that for broken state we want only notify the user (as described in the comment). My attempts to shutdown domain from inside leads to the halted state.
Halt from inside do not lead to any event at all and just hung the VM in running state, but it looks like the separate issue.